conans/client/command.py

```diff
@@ -311,7 +311,8 @@ def info(self, *args):
                             'e.g., MyPackage/1.2@user/channel or ./my_project/')
         parser.add_argument("--file", "-f", help="specify conanfile filename")
         parser.add_argument("--only", "-n", nargs="?", const="None",
```

Comment:
--paths option is not admitting values:

conan info --paths export_folder

ERROR: Conanfile not found!